cli/cli/gateway.py

```diff
@@ -115,23 +122,39 @@ def print_routes(self) -> None:
         routes: list[Route] = self.get_routes()
         routes.sort(key=lambda r: r.relative_url)
 
-        table = Table("Relative url", "Target", "HTTP methods")
+        table = Table("Relative url", "Target", "HTTP methods", "JWT", "CORS")
         for route in routes:
+            jwt = "On" if route.jwt else "Off"
+            cors = "On" if route.cors else "Off"
```

Comment:
Could also use an emoji for this like :x: and :heavy_check_mark: haha

Reply:
Haha yes I did consider this but certain shells don't support emojis and it can be very annoying for those users as it will randomly kill their session.

Reply:
`-` and `On` are much clearer though so I will go with that